cloudify_aws/workflows/tests/test_utils.py

```diff
@@ -0,0 +1,56 @@
+from mock import patch, call
+from unittest import TestCase
+
+from ...common import utils
+
+
+class AWSWorkflowUtilsTests(TestCase):
+
+    def test_desecretize_client_config(self):
```

Comment:
same. if we move the utils code to utilities_sdk so we can move these tests too.